sources/whippet/src/main.rs

```diff
@@ -0,0 +1,135 @@
+/*!
+# Introduction
+
+*whippet* is Bottlerocket's implementation of the dbus-launcher. It implements the minimal
```

Comment:
`dbus-launcher` is not the name of the program you're replacing.
```suggestion
*whippet* is Bottlerocket's implementation of a launcher for `dbus-broker`. It implements the minimal
```